crates/journal-client/src/fragments.rs

```diff
@@ -49,4 +50,18 @@ impl FragmentIter {
         }
         next.map(|r| Ok(r))
     }
+
+    pub fn as_stream<'a>(&'a mut self) -> impl TryStream<Ok = broker::fragments_response::Fragment, Error = Error> + 'a {
```

Comment:
I haven't been living in Rust land to intuit the consequences of that. My very vague understanding was to stay away from `'static` because it provides less flexibility, but I don't actually know why that's true or when to apply that rule.

Reply:
Done